Makefile

```diff
@@ -34,3 +34,32 @@ coverage: test ## Run unit tests and show code coverage.
 docker-build: ## Build the docker image.
 	docker build -f Dockerfile . -t search-v2-api
 
+N_USERS ?=2
+HOST ?= $(shell oc get route search-api -o custom-columns=host:.spec.host --no-headers -n open-cluster-management --ignore-not-found=true)
+ifeq ($(strip $(HOST)),)
+	CONFIGURATION_MSG = @echo \\n\\tThe search-api route was not found in the target cluster.\\n\
+	\\tThis test will run against the local instance https://localhost:4010\\n\
+	\\tIf you want to run this test against a cluster, create the route with make test-scale-setup\\n;
+	
+	HOST = localhost:4010
+endif
+export API_TOKEN :=$(shell oc whoami -t)
+
+test-scale-setup: ## Creates the search-api route in the current target cluster.
+	oc create route passthrough search-api --service=search-search-api -n open-cluster-management
+
+test-scale: check-locust ## Sends multiple simulated requests for testing using Locust. Use N_USERS to change the number of simulated users.
```

Comment:
can we specify in comment if this is per cluster or multiple.

Reply:
Updated this comment, let me know if it's not clear.

We are simulating users logged into ACM and using search.  These will be independent of the number of clusters managed by ACM.